### PR TITLE
Feat: Added Questions page

### PR DESCRIPTION
**Pull request by sainath7878**
Added question page

Files to review

- [src/App.js](https://github.com/sainath7878/joddquiz/pull/3/files#diff-3d74dddefb6e35fbffe3c76ec0712d5c416352d9449e2fcc8210a9dee57dff67)
- [src/components/questionCard/QuestionCard.jsx](https://github.com/sainath7878/joddquiz/pull/3/files#diff-268633646358a354c1889d801862312a486c58b483ce180df6d2adbce51b5a74)
- [src/components/questionCard/questionCard.module.css](https://github.com/sainath7878/joddquiz/pull/3/files#diff-96fe78cd9b3540aacbcb7f1d908c2f5650e717a892018b510fcb96aea12fced5)
- [src/context/quizContext.js](https://github.com/sainath7878/joddquiz/pull/3/files#diff-17b6bea987918f8760f5768960eeb15145da8b073ce55af5e6ab7819de1b3be4)
- [src/pages/quiz/Quiz.jsx](https://github.com/sainath7878/joddquiz/pull/3/files#diff-f358fe19efb423c155fbacbf372f40cfa4174645c2c38a70c50042535901ab6d)
- [src/reducer/quizReducer.js](https://github.com/sainath7878/joddquiz/pull/3/files#diff-cb3fe1c5da553865be877bdbeaf374760d3abdadb82e86ac952f0adfeeff1bc1)

### Thanks for reviewing!

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -9,6 +9,11 @@
 }
 
 .loader {
+  /* position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%); */
```
Comment (anik31) [inline src/App.css:15]: suggestion - remove commented code

**Review by Maianki — APPROVED** [review]
Clean code structure and great namings. Good to merge.

File: src/App.js [hunk 2]
```diff
@@ -28,6 +28,12 @@ function App() {
             <Route path="/rules" >
               <Route path=":quizid" element={<RulesPage />} />
             </Route>
+            <Route path="/questions">
+              <Route path=":quizid" >
+                <Route path=":questionNumber" element={<Quiz />} />
+              </Route>
+            </Route>
```
Comment (anik31) [inline src/App.js:35]: appreciation - good that you're having nested routes

**Review by anik31 — APPROVED** [review]
Everything looks good. Nice work.

File: src/App.css (hunk 1 above)
Comment (sainath7878) [reply src/App.css:15]: Yeah, will do. Thanks for pointing out!

**Pull request merged**